media-video/mpv/mpv-0.35.1.recipe

```diff
@@ -130,6 +133,22 @@ INSTALL()
 	rm -rf $dataDir/applications
 	rm -rf $dataDir/icons
 
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local SHORT_INFO=$portName
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@SHORT_INFO@|$SHORT_INFO|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/$portName.rdef.in > $portName.rdef
+
+	addResourcesToBinaries $portName.rdef $binDir/$portName
+	mimeset -f $binDir/$portName
```

Comment:
Have you tested this on 32 bit? `$portName` will be `mpv_x86` so I'm not sure some of these commands will work.  Specifically the `$portDir/additional-files/$portName.rdef.in` and  the two with `$binDir/$portName`.  I haven't had a chance to try it out though.